ql/src/test/org/apache/hadoop/hive/metastore/txn/TestTxnHandler.java

```diff
@@ -1692,7 +1692,7 @@ public void testReplOpenTxn() throws Exception {
     int numTxn = 50000;
     String[] output = TestTxnDbUtil.queryToString(conf, "SELECT MAX(\"TXN_ID\") + 1 FROM \"TXNS\"").split("\n");
     long startTxnId = Long.parseLong(output[1].trim());
-    txnHandler.setOpenTxnTimeOutMillis(30000);
+    txnHandler.setOpenTxnTimeOutMillis(60000);
```

Comment:
This timeout (failing test) was giving me a lot of heartburn.  I have been playing around with it on `master` branch and it's pretty sensitive to change.  Lowering it just a little causes timeout failures.  I have bumped it here to get it to pass and to make this test less flaky.